Last-Update: 2015-06-16
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: set speaker "off" by default
 Do not forward speaker on attach; sound forwarding is to be enabled from
 menu so it won't slow down session on slow link or play unexpectedly.

--- a/etc/xpra/conf.d/20_sound.conf.in
+++ b/etc/xpra/conf.d/20_sound.conf.in
@@ -1,11 +1,11 @@
 ################################################################################
 # Sound
 
 # Forward sound output to clients:
-#speaker = off
+speaker = off
 #speaker = disabled
-speaker = on
+#speaker = on
 
 # Forward sound input to server:
 # microphone = disabled
 # microphone = on
